.github/workflows/e2e-pr.yml

```
@@ -92,16 +92,47 @@ jobs:
                   MAX_RETRIES: 3
                   RETRY_DELAY: 10000
 
+              # Properties specified in the body for MRT API to update settings must be in MRT_ENV_SETTINGS_E2E_BASE_FILENAME file.
+              # See: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/references/mrt-admin?meta=projects_target_partial_update
+              # Like proxy configs, enabling cookies or other MRT env settings.
+            - name: Create .env with settings to be updated on the MRT target
+              run: |-
+                  echo "MRT_TARGET_SSR_PROXY_CONFIGS=${{ vars.PWA_KIT_E2E_TARGET_PROXY_CONFIGS }}" >> ${{ vars.MRT_ENV_SETTINGS_E2E_BASE_FILENAME }}
+
+              # Environment variables you'd like to set on the MRT target must be in MRT_ENV_VARS_E2E_BASE_FILENAME file.
+              # See: https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/references/mrt-admin?meta=projects_target_env_var_partial_update
+              # These are the env variables required by the storefront to be set in MRT for the e2e tests to run.
+              # Like SLAS client secret.
+            - name: Create .env with environment variables to be set on the MRT target
+              run: |-
+                  echo "PWA_KIT_SLAS_CLIENT_SECRET=${{ secrets.ZZRF_002_SLAS_PRIVATE_CLIENT_SECRET }}" > ${{ vars.MRT_ENV_VARS_E2E_BASE_FILENAME }}
```

To be consistent with the previous step, use double arrows? And perhaps add a comment saying that initially the file is empty.

```suggestion
                  echo "PWA_KIT_SLAS_CLIENT_SECRET=${{ secrets.ZZRF_002_SLAS_PRIVATE_CLIENT_SECRET }}" >> ${{ vars.MRT_ENV_VARS_E2E_BASE_FILENAME }}
```

---

I actually changed the other one to '>' since '>' means create while '>>' means append.